src/mon/MDSMonitor.cc

```diff
@@ -1557,6 +1557,16 @@ bool MDSMonitor::has_one_of_these_health_warnings(vector<mds_metric_t> warnings)
   return false;
 }
 
+bool MDSMonitor::has_health_warnings()
```

Comment:
nit: `has_any_health_warning()` and keep `has_health_warnings` as it is.

Reply:
I originally thought the same the but I perfer current version as I find it clearer.

Reply:
An earlier commit changes has_health_warnings to has_one_of_these_health_warnings and this commit introduces has_health_warnings. I don't understand why. Suggest change this to has_any_health_warning and not change the original has_health_warnings.

Reply:
Unlike `has_health_warnings()`, `has_one_of_these_health_warnings()` tells that this method looks for specific health warnings that have will be passed by the caller. I like this emphasis since this is more precise and makes code succinct and thereby easier to read.

Reply:
This is fine if its pythonic code where the arg tells nothing about its type -- even that's changing now with `Typing`. Unless the function name does not make any sense *at all*, I'd appreciate it to be left alone.

Reply:
okay.

Reply:
Made that change. Marking this as resolved.